nautobot/extras/tests/test_api.py

```diff
@@ -1864,3 +1865,89 @@ def setUpTestData(cls):
         for webhook in webhooks:
             webhook.save()
             webhook.content_types.set([obj_type])
+
+    def test_create_webhooks_with_diff_content_type_same_url_same_action(self):
+        """
+        Create a new webhook with diffrent content_types, same url and same action with a webhook that exists
+
+        Example:
+            Webhook 1: dcim | device type, create, http://localhost
+            Webhook 2: dcim | console port, create, http://localhost
+        """
+        obj_perm = ObjectPermission(name="Test permission", actions=["add"])
+        obj_perm.save()
+        obj_perm.users.add(self.user)
+        obj_perm.object_types.add(ContentType.objects.get_for_model(self.model))
+
+        data = (
+            {
+                "content_types": ["dcim.consoleport"],
+                "name": "api-test-7",
+                "type_create": True,
+                "payload_url": "http://api-test-1.com/test1",
+                "http_method": "POST",
+                "http_content_type": "application/json",
+                "ssl_verification": True,
+            },
+        )
+
+        response = self.client.post(self._get_list_url(), data, format="json", **self.header)
+        self.assertHttpStatus(response, status.HTTP_201_CREATED)
+
+    def test_create_webhooks_with_same_content_type_same_url_diff_action(self):
+        """
+        Create a new webhook with same content_types, same url and diff action with a webhook that exists
+
+        Example:
+            Webhook 1: dcim | device type, create, http://localhost
+            Webhook 2: dcim | device type, delete, http://localhost
+        """
+        obj_perm = ObjectPermission(name="Test permission", actions=["add"])
+        obj_perm.save()
+        obj_perm.users.add(self.user)
+        obj_perm.object_types.add(ContentType.objects.get_for_model(self.model))
+
+        data = (
+            {
+                "content_types": ["dcim.devicetype"],
+                "name": "api-test-7",
+                "type_update": True,
+                "payload_url": "http://api-test-1.com/test1",
```

Comment:
It occurs to me that we shouldn't be using potentially "real" URLs in any of these tests. Would you please change all of these to either `example.com/...` or `example.invalid/...` throughout?